sitemedia/js/components/FilterChoice.js

```diff
@@ -0,0 +1,25 @@
+export default Vue.component ('FilterChoice', {
+    template: `
+    <label :active="active" is="sui-button" role="checkbox" basic circular compact>
```

Comment:
so the `is="sui-button"` tells Vue that I'm using a different tag (`label`) when I actually want to render a button (`sui-button`), which would receive the `basic circular compact` boolean attributes at the end of the tag as props to `sui-button`, which ultimately renders:

```html
<label class="ui button basic circular compact"></label>
```

this is coming from [semantic-ui-vue](https://semantic-ui-vue.github.io/#/). we can talk more about the library - it seems like it might not be well-maintained, but I'm not sure if there is an alternative to make the semantic UI behaviors play well with Vue.